teacher_class.rb

```diff
@@ -0,0 +1,16 @@
+require './person_class'
+# Teacher class
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(
+    specialization, _age, _name = 'Unknown', parent_permission = 'true'
```

Comment:
- [ ] Same as above find away to avoid the underscore because it returns a nil value instead of the inherited value from a person.

Reply:
Thank you for comments, I have deleted the underscores